shell/common/tracing_controller.cc

```diff
@@ -30,6 +30,8 @@ static void AddTraceMetadata() {
   blink::Threads::Gpu()->PostTask([]() { Dart_SetThreadName("gpu_thread"); });
   blink::Threads::UI()->PostTask([]() { Dart_SetThreadName("ui_thread"); });
   blink::Threads::IO()->PostTask([]() { Dart_SetThreadName("io_thread"); });
+  blink::Threads::Platform()->PostTask(
```

Comment:
This name appears on the timeline. We are not modifying the actual thread name (and cause clients to be confused when their thread names are changed from under them).

On Fuchsia, the timeline will say platform_thread as that is the last task posted. I can make it say ui_thread if you want.